druid-shell/src/backend/windows/window.rs

```diff
@@ -237,6 +237,7 @@ struct WindowState {
     // Is the window focusable ("activatable" in Win32 terminology)?
     // False for tooltips, to prevent stealing focus from owner window.
     is_focusable: bool,
+    window_level: RefCell<Option<WindowLevel>>,
```

Comment:
This never changes, right? So you don't need the `RefCell`. And I think `AppWindow` is a good default so we don't need the `Option` either.

Reply:
Dang! Forgot to revert this after removing `set_level`. Thanks.